### PR TITLE
SNOW-875973: Prepare nodejs centos7 node14 docker image

### PR DESCRIPTION
**Pull request by sfc-gh-dprzybysz**
### Description
Add centos7 with the latest node 14 docker images for build and test (they are equal for the internal CI process needs) 

### Checklist
- [ ] Format code according to the existing code style (run `npm run lint:check -- CHANGED_FILES` and fix problems in changed code)
- [ ] Create tests which fail without the change (if possible)
- [ ] Make all tests (unit and integration) pass (`npm run test:unit` and `npm run test:integration`)
- [ ] Extend the README / documentation and ensure is properly displayed (if necessary)
- [ ] Provide JIRA issue id (if possible) or GitHub issue id in commit message

### CODE REVIEW

File: ci/image/Dockerfile.nodejs-centos7-node14-test [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.4/install.sh | bash && \
+    source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+COPY scripts/npmrc /root/.npmrc
+RUN npm install npm@latest -g
```
Comment (sfc-gh-pmotacki) [inline ci/image/Dockerfile.nodejs-centos7-node14-test:24]: Do we want to use latest npm ?

File: ci/image/Dockerfile.nodejs-centos7-node14-build [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.4/install.sh | bash && \
+    source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+COPY scripts/npmrc /root/.npmrc
+RUN npm install npm@latest -g
```
Comment (sfc-gh-pmotacki) [inline ci/image/Dockerfile.nodejs-centos7-node14-build:24]: Do we want to use latest npm ?

**Review by sfc-gh-pmotacki — APPROVED** [review]
I've added only some questions to be sure that we would like to use always latest npm.

File: ci/image/Dockerfile.nodejs-centos7-node14-test (hunk 1 above)
Comment (sfc-gh-dprzybysz) [reply ci/image/Dockerfile.nodejs-centos7-node14-test]: deleted

File: ci/image/Dockerfile.nodejs-centos7-node14-build (hunk 2 above)
Comment (sfc-gh-dprzybysz) [reply ci/image/Dockerfile.nodejs-centos7-node14-build]: deleted

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Pull request merged**